python-client/giskard/scanner/robustness/base_detector.py

```diff
@@ -134,26 +134,26 @@ def _detect_issues(
                 # Severity
                 issue_level = IssueLevel.MAJOR if fail_rate >= 2 * threshold else IssueLevel.MEDIUM
 
-                # Prepare examples dataframe
-                examples = original_data.df.loc[~passed, (feature,)].copy()
-                examples[f"{transformation_fn.name}({feature})"] = perturbed_data.df.loc[~passed, feature]
-                examples["Original prediction"] = original_pred.prediction[~passed]
-                examples["Prediction after perturbation"] = perturbed_pred.prediction[~passed]
-
                 # Description
                 desc = "When we perturb the content of feature “{feature}” with the transformation “{transformation_fn}” (see examples below), your model changes its prediction in about {fail_rate_percent}% of the cases. We expected the predictions not to be affected by this transformation."
 
+                failed_size = (~passed).sum()
+                slice_size = len(passed)
+
                 issue = Issue(
                     model,
                     dataset,
                     group=self._issue_group,
                     level=issue_level,
                     transformation_fn=transformation_fn,
                     description=desc,
+                    features=[feature],
                     meta={
                         "feature": feature,
                         "domain": f"Feature `{feature}`",
-                        "deviation": f"{round(fail_rate * 100, 2)}% of tested samples changed prediction after perturbation",
+                        "deviation": f"{failed_size} out of {slice_size} tested samples ({round(fail_rate * 100, 2)}%) changed prediction after perturbation",
```

Comment:
we should unify the convention here, I used `{failed_size}/{slice_size}` in the prompt-injection.

Reply:
Let’s keep what is the most clear in your opinion